components/BookmarkButton/index.js

```diff
@@ -0,0 +1,22 @@
+import Heart from "../../assets/heart.svg";
+import styled from "styled-components";
+
+const StyledButton = styled.button`
+  background-color: ${({ $isBookmarked }) =>
```

Comment:
you can use a ternary operator in the button component prop 'isBookmarked' directly on line 15 here.